### ISSUE TITLE
Paper release

### ISSUE THREAD

**Issue by vadimkantorov** [chenbinghui1/DSL#1]
http://www4.comp.polyu.edu.hk/~cslzhang/paper/DSL_cvpr22.pdf was linked from Google Scholar but unfortunately returns 404 now. Would be keen to read the paper in better rendering than Google cache :)

**Comment by chenbinghui1** [comment]
@vadimkantorov Hi, Tanks for attention to our work. This work belongs to Alibaba group and the license is now being approved. The code and paper will be released after approval.

**Comment by vadimkantorov** [comment]
About paper: quite interesting given that the camera-ready deadline has passed :)

**Comment by chenbinghui1** [comment]
@vadimkantorov Deadline of camera-ready has no relation to the approval of company. Each paper published by Alibaba must be approved by company itself. If not the staff will violate company rules. Sorry for that.

**Comment by vadimkantorov** [comment]
Camera-ready submission has relation with paper release. During camera-ready submission IIRC you approve copyright waiver to IEEE and CV Foundation. If the paper is not retracted, it will be published in the proceedings and available online anyway - just a few months later. So if Alibaba does not give approvbal to paper release, I guess it should be retracted. So this is the reason of my confusion.

**Comment by chenbinghui1** [comment]
@vadimkantorov Indeed, the paper will be online available and the release will also be finally approved by Alibaba. But it is not now, and the only thing we can do is just wait. From the deadline of camera-ready to the date of online release of CVPR there are enough days for the company to deal with this approval. I know your confusion, so please wait together. Thanks~

**Comment by vadimkantorov** [comment]
thanks for the release!
